tamperMonkey/CTC-Departures-Section-Auto-Completer.user.js

```diff
@@ -137,7 +137,7 @@ function initialiseJourneys() {
                 new ButtonPage("/manage-transit-movements/departures/transport-details/authorisations/add/", "value-no"),
                 new InputPage("/manage-transit-movements/departures/transport-details/authorisations/1/reference-number/", "TRD123"),
                 new ButtonPage("/manage-transit-movements/departures/transport-details/authorisations/add-another/", "value-no"),
-                new InputPage("/manage-transit-movements/departures/transport-details/carrier/eori-tin/", "carrierEORI123"),
+                new InputPage("/manage-transit-movements/departures/transport-details/carrier/eori-tcuin/", "carrierEORI123"),
```

Comment:
`carrierEORI123` is not a valid value. Which is interesting, because it looks like it should be given the regex we've implemented. However, going over the original error again (https://hmrcdigital.slack.com/archives/CPN5YJ62U/p1686663549471369), it looks like the first two characters should be uppercase? Can you chase this up with Sayak please @kenneth-rayner ?

Reply:
Going off the origin code in the ticket:
"n1:TINNewType":
{     "description": "EORI or TCUI Number (format: an..17)",     "maxLength": 17,     "minLength": 3,     "pattern": "^([A-Z]
{2,2}
\\!-~
{1,15}
)$",
     "type": "string"
   },
   it does show [A-z] not [A-Za-z] 
   I have updated the regex in Transit FE regex and also the above line 140 to input a value starting with caps only. Sayak has agreed